ci/workflows/pull-request.yml

```diff
@@ -79,7 +79,7 @@ steps:
           config: ci/docker-compose.yml
           mount-buildkite-agent: true
       - ./ci/plugins/upload-failure-logs
-    timeout_in_minutes: 10
+    timeout_in_minutes: 30 # This timeout accommodates PRs with Sqlsmith job running.
```

Comment:
Is it necessary to do a fuzzing test in every PR? Doing so will quickly hit the buildkite job minutes limit.

Reply:
No, not every PR will run the fuzzing test

Reply:
This timeout is for PRs which have to run it, i.e. PRs which modify sqlsmith ci / src files.

Reply:
ok, it turns out that the fuzzing test will only be run in pr when the following files are modified.
```
MATCHES="ci/scripts/pr.env.sh\
\|ci/scripts/run-e2e-test.sh\
\|ci/scripts/run-unit-test.sh\
\|src/tests/sqlsmith"
```